app/views/middleware_topology/show.html.haml

```diff
@@ -0,0 +1,32 @@
+%div{'data-ng-app' => "mwTopologyApp", 'ng-controller' => "middlewareTopologyController"}
+  = stylesheet_link_tag 'container_topology'
```

Comment:
This is the only reason you need `container_topology` in `assets.rb`. Remove this line and you can remove that one too.

Also, note that most of the rules in `container_topology.css` now requires a `.container_topology` root element so you may need it here.